streamer.inc

```diff
@@ -357,6 +357,7 @@ native IsValidDynamicActor(STREAMER_TAG_ACTOR actorid);
 native IsDynamicActorStreamedIn(STREAMER_TAG_ACTOR actorid, forplayerid);
 native GetDynamicActorVirtualWorld(STREAMER_TAG_ACTOR actorid);
 native SetDynamicActorVirtualWorld(STREAMER_TAG_ACTOR actorid, vworld);
+native GetDynamicActorAnimation(STREAMER_TAG_ACTOR actorid, animlib[], animname[], &Float:fdelta, &loop, &lockx, &locky, &freeze, &time, maxanimlib = sizeof animlib, maxanimname = sizeof animname);
```

Comment:
They aren't const, they are modified by the plugin. const should be added only when that parameter won't be modified under any circumstance.